doc/api/stream.md

```diff
@@ -1706,8 +1704,14 @@ objects and passing appropriate methods as constructor options.
 const { Writable } = require('stream');
 
 const myWritable = new Writable({
+  construct(options, callback) {
+    // Initialize state and load resources...
+  },
   write(chunk, encoding, callback) {
     // ...
+  },
+  destroy() {
```

Comment:
This should be paired with the `construct` method, just like `construct/destruct`, `init/deinit` and `create/destroy`.

Reply:
It is? I'm not sure what you mean here.